### ISSUE TITLE
AttributeError from print statements using Python (PyImageJ) in Fiji

### ISSUE THREAD

**Issue by karlduderstadt** [scijava/scyjava#65]
I have been testing the Python (PyImageJ) scripting language in Fiji using jaunch together with Java 21. The following script:
```python
print('hi')
```
results in the following error:
```python
Traceback (most recent call last):
  File "/Users/karlduderstadt/miniconda3/envs/pyimagej/lib/python3.11/site-packages/scyjava/_script.py", line 100, in apply
    return_value = eval(
                   ^^^^^
  File "<string>", line 1, in <module>
  File "/Users/karlduderstadt/miniconda3/envs/pyimagej/lib/python3.11/site-packages/scyjava/_script.py", line 47, in write
    self._writer().write(s)
    ^^^^^^^^^^^^^^^^^^^^
AttributeError: 'org.scijava.script.AbstractScriptContext' object has no attribute 'write'
```
I suspect this issue has been around since the python script runner moved from pyimagej to scyjava, but haven't tested it in a long while.

Basically, we need to call `.getWriter()` on `AbstractScriptContext` to get the actual writer object that has the write method. Somehow this was lost in the migration. Lines 49 - 52 need to be changed to
```python
def _writer(self):
            return self._thread_to_context.get(
                threading.currentThread(), self._std_default
            ).getWriter()
```
this fixed the issue in my testing. I will try and make a PR...

**Comment by karlduderstadt** [comment]
oh I think this is fixed now on the main branch... I was working with the current version just through conda...

**Comment by ctrueden** [comment]
@karlduderstadt Thanks for testing and for the report! Yes, I think I encountered this same problem when I tested recently, and fixed it with 5f5a957921199d651912a03435c8cd1239c1845d. Sorry I forgot about this! We need to get a new release of scyjava (and a new release of pyimagej) done soon.
